pkgs/by-name/ge/gema/package.nix

```
@@ -0,0 +1,35 @@
+{ lib
+, stdenv
+, fetchurl
+, installShellFiles
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gema";
+  version = "2.0";
+
+  src = fetchurl {
+    url = "https://sourceforge.net/projects/gema/files/gema/${finalAttrs.pname}-${finalAttrs.version}/${finalAttrs.pname}-${finalAttrs.version}-source.tar.gz";
```

Please avoid superfluous use of `pname` (see #277994 for details):
```suggestion
    url = "https://sourceforge.net/projects/gema/files/gema/gema-${finalAttrs.version}/gema-${finalAttrs.version}-source.tar.gz";
```

---

Thanks! After reading #27994 it makes sense that pname shouldn't be used in the url. For example, it could be reasonable to want to install two different versions of a package (say gema) each with different names and different patches applied. Changing the pname would prevent this.